docs/README.md

```diff
@@ -1,15 +1,30 @@
----
-editLink: true
```

Comment:
Looks like you didn't pull recent changes before commit. This front-matter should stay if we don't want dead links.